# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../electricfence/no-tests.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

No tests (yet), since our flist wrapper is causing headache.

  - Rene Rebe <rene@exactcode.de>

--- efence/Makefile.orig	2005-02-11 03:10:28.000000000 +0100
+++ efence/Makefile	2005-02-13 14:33:50.576793840 +0100
@@ -54,11 +54,7 @@
 OBJECTS = efencepp.o efence.o
 
 all:	libefence.a $(EFENCESO) tstheap eftest eftestpp
-	@ echo "Testing Electric Fence."
-	@ echo "After the last test, it should print that the test has PASSED."
-	$(CURPATH)eftest
-	$(CURPATH)tstheap 3072
-	@ echo "Electric Fence confidence test PASSED."
+	@ echo "Testing Electric Fence skipped."
 
 install: libefence.a efence.3 libefence.so.0.0
 	$(INSTALL) -m 755 ef.sh $(BIN_INSTALL_DIR)/ef
